interface/frontend/src/components/Nodule.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+<div class="card">
+  <div class="card-header">
+    <p class="mb-0">
+      <a @click="toggleShow(nodule)">
+        <b>Nodule {{ index + 1 }}</b>
+      </a>
+    </p>
+  </div>
+
+  <div class="collapse" :class="{ show: isOpen }">
+    <div class="card-block">
+      <table>
+        <tbody>
+        <tr>
+          <td>
+            <select v-model="selected">
+              <option disabled> -- select an option -- </option>
+              <option :value="lungOrientations.left">Left lung</option>
+              <option :value="lungOrientations.right">Right lung</option>
+            </select>
+          </td>
+        </tr>
+        </tbody>
+      </table>
+      <br>
+      <a @click="update(nodule)">
+        <button type="button" class="btn btn-sm btn-secondary">Accept</button>
+      </a>
+    </div>
+  </div>
+</div>
+</template>
+
+<script>
+export default {
+  props: ['nodule', 'index'],
+  data () {
+    return {
+      isOpen: false,
+      selected: this.nodule.lung_orientation,
+      lungOrientations: this.$constants.lungOrientations
+    }
+  },
+  created () {
+    if (this.index === 0) { this.isOpen = true }
+  },
+  methods: {
+    toggleShow (nodule) {
+      this.isOpen = !this.isOpen
+    },
+    update (nodule) {
+      this.$axios.put(nodule.url, { lung_orientation: this.selected }).then((response) => {
```

Comment:
FYI, at the moment `PUT` requests are failing because of [Preflighted requests](https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS#Preflighted_requests). They are causing an `OPTION` request to occur which fails likely because the backend API isn't set up to respond to that type of request. I believe the solution is to allow CORS on the backend.

Reply:
Ok correction: looks like `OPTIONS` requests are actually allowed, but `PUT` is not.

<img width="317" alt="screen shot 2017-09-18 at 8 55 55 pm" src="https://user-images.githubusercontent.com/7776175/30573981-3c228d7a-9cb4-11e7-9a59-3990319bedb1.png">